app/pods/application/template.hbs

```diff
@@ -1,10 +1,10 @@
 <!-- Overlay For Sidebars -->
 <div class="overlay"></div>
 <!-- #END# Overlay For Sidebars -->
-
-{{#nav-bar model=model}}{{/nav-bar}}
-
-{{#left-side-bar model=model}}{{/left-side-bar}}
+{{#unless embeded}}
```

Comment:
@aayusharora  An example on how to use the flag, we will not show navbar when in embed mode. ( or We will show navbar unless inside an iFrame )